requirements.txt

```diff
@@ -3,6 +3,8 @@ h5py >= 3.0.0
 numpy >= 1.20.0
 scipy >= 1.3.0
 pytest
+pyblock
 pandas == 1.5.1 # issue with pyblock and pandas > 2
 mpi4py >= 3.0.0
-numba
\ No newline at end of file
+numba
+trexio
```

Comment:
Remove this as a requirement please. It should be optional.

Reply:
Remove pyblock too, because we are not even testing that anymore.

Reply:
Done